parts/k8s/kubernetesparams.t

```diff
@@ -1,3 +1,31 @@
+    "planName": {
```

Comment:
I don't understand why these new parameters are being added and we can't use osImageOffer, osImageSKU, etc?

Reply:
I don't think `plan` should be set unconditionally in VMs. I think we need to explicit about when we use it.

Reply:
Well, for image references we would avoid setting plan...